src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -901,6 +903,20 @@ private boolean isEmptyCtorBlock(DetailAST ast, int parentType) {
                     || isEmptyBlock(ast, parentType, TokenTypes.COMPACT_CTOR_DEF));
     }
 
+    /**
+     * Test if the given {@code DetailAST} is a part of an
+     * empty constructor.
+     *
+     * @param ast the {@code DetailAST} to test.
+     * @return {@code true} if {@code ast} makes up part of an
+     *          empty constructor block.
+     */
+    private static boolean isEmptyCtorBlock(DetailAST ast) {
+        return (ast.getParent().getType() == TokenTypes.CTOR_DEF
+                || ast.getParent().getType() == TokenTypes.COMPACT_CTOR_DEF)
```

Comment:
+4 spaces in indentation, to visually see priority of execution

Reply:
done